benchto-driver/src/test/resources/application.yaml

```diff
@@ -1,5 +1,5 @@
 benchmarks: src/test/resources/benchmarks
-sql: src/test/resources/sql
+sql: src/test/resources/sql,src/test/resources/sql-second-dir
```

Comment:
order of properties: http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html
System properties take precedence over yaml files